backpack.tf extended sorting.user.js

```diff
@@ -1268,4 +1268,246 @@ class</a></li>
+                if(this.idreplacedict.hasOwnProperty(defindexInt)){
+                    itemId = this.idreplacedict[defindexInt].toString();
+                }
+                else{
+                    itemId = this.defindex;
+                }
+            }
+            return itemId;
+        }
+
+        getItemSlot(){
+            let itemSlot;
+            switch(this.slot){
+                case "primary": case "secondary": case "melee": case "pda":
+                    itemSlot = 1;
+                    break;
+                case "misc":
+                    itemSlot = 0;
+                    break;
+                case "none":
+                    if(this.name == "Kit"){
+                        itemSlot = -1;
+                    }
+                    else if(this.name == "Fabricator"){
+                        itemSlot = -2;
+                    }
+                    else{
+                        itemSlot = 0;
+                    }
+                    break;
+            }
+            return itemSlot;
+        }
+
+        getKillstreakTier(){
+            return this.killstreak_tier;
+        }
+    };
+
+    if (window.location.pathname.startsWith("/stats")) {
+        const currentItem = new itemData(document.querySelector(".item"));
+        let itemId = currentItem.getItemId();
+        let killstreakTier = currentItem.getKillstreakTier();
+        let currentPath = window.location.pathname;
+
+        $(".stats-header-controls").append(`
+     <br>
+     <div class="btn-group btn-group-sm stats-killstreak-list" style="margin: 0 0 12px"></div>
+     `);
+
+        switch(currentItem.getItemSlot()){
+
+            case 0:
+                break;
+
+            case -1: {
+
+                let weaponName = trimSpaces(document.querySelector(".stats-breadcrumbs").children[1].innerHTML);
+                let weaponDefIndex = window.location.pathname.match(/\d+$/g);
+
+                $(".stats-killstreak-list").append(`
+          <a class="btn btn-variety q-440-text-4" href="https://backpack.tf/stats/Unique/Killstreak%20Kit/Tradable/Non-Craftable/1-${weaponDefIndex}"> Basic </a>
+     <a class="btn btn-variety q-440-text-7" href="https://backpack.tf/stats/Unique/Killstreak%20Kit/Tradable/Non-Craftable/2-${weaponDefIndex}"> Specialized </a>
+     <a class="btn btn-variety q-440-text-11" href="https://backpack.tf/stats/Unique/Killstreak%20Kit/Tradable/Non-Craftable/3-${weaponDefIndex}"> Professional </a>
+     `);
+                document.querySelector(".stats-killstreak-list").children[killstreakTier-1].classList.add("active");
+                if(killstreakTier > 1){
+                    $(".stats-header-controls").append(`
+     <a class="btn btn-default" href="https://backpack.tf/stats/Unique/Fabricator/Tradable/Craftable/${6520 + 3 * (killstreakTier - 1)}-6-${weaponDefIndex}" style="margin: 0px 0px 12px;">
+     <i class="fa fa-bar-chart"></i>
+     Fabricator Stats
+     </div>
```

Comment:
```suggestion
     </a>
```

Reply:
You open an anchor tag, but close a div tag, the same is further below too if I remember correctly, I just forgot to make a suggestion.